src/index.js

```diff
@@ -0,0 +1,38 @@
+import i18n from "./i18n/index.js";
+import schemas from "./schemas/index.js";
+import exampleCapturePayment from "./util/exampleCapturePayment.js";
+import exampleCreateAuthorizedPayment from "./util/exampleCreateAuthorizedPayment.js";
+import exampleCreateRefund from "./util/exampleCreateRefund.js";
+import exampleListRefunds from "./util/exampleListRefunds.js";
+import startup from "./startup.js";
+
+/**
+ * @summary Import and call this function to add this plugin to your API.
+ * @param {ReactionAPI} app The ReactionAPI instance
+ * @returns {undefined}
+ */
+export default async function register(app) {
+  await app.registerPlugin({
+    label: "ExamplePayment",
+    name: "example-paymentmethod",
+    version: app.context.appVersion,
```

Comment:
This version should be imported from `package.json`.

Change this line to `version: pkg.version`, and then `import pkg from "../package.json";`